src/renderer/components/ft-select/ft-select.vue

```diff
@@ -28,7 +28,7 @@
     <label
       class="select-label"
       :for="sanitizedId ?? sanitizedPlaceholder"
-      :hidden="disabled"
+      v-if="!disabled"
```

Comment:
Say hi to warning

Reply:
? Cannot replicate. Can you take a screenshot?

Reply:
https://github.com/FreeTubeApp/FreeTube/pull/4970/files#diff-04846a18b5ef86834f2cbee911c63a2f616a0e4f68d8f269ffaaa3e5051ad8b3R31

![2024-04-26 10_47_56-Window](https://github.com/FreeTubeApp/FreeTube/assets/1018543/88c871d4-763e-4d6c-b900-010900b4572e)

Reply:
:facepalm: I thought you mean in the dev console. My bad lol